README.md

```diff
@@ -36,29 +36,34 @@ $ ./cql-proxy -h
+                                                          ($ASTRA_BUNDLE).
+  -t, --astra-token=STRING                                Token used to authenticate to an Astra database. Requires '--astra-database-id'. Ignored if using the bundle path or contact points option
+                                                          ($ASTRA_TOKEN).
+  -i, --astra-database-id=STRING                          Database ID of the Astra database. Requires '--astra-token' ($ASTRA_DATABASE_ID)
+      --astra-api-url="https://api.astra.datastax.com"    URL for the Astra API ($ASTRA_API_URL)
+  -c, --contact-points=CONTACT-POINTS,...                 Contact points for cluster. Ignored if using the bundle path or token option ($CONTACT_POINTS).
+  -u, --username=STRING                                   Username to use for authentication ($USERNAME)
+  -p, --password=STRING                                   Password to use for authentication ($PASSWORD)
+  -r, --port=9042                                         Default port to use when connecting to cluster ($PORT)
+  -n, --protocol-version="v4"                             Initial protocol version to use when connecting to the backend cluster (default: v4, options: v3, v4, v5, DSEv1, DSEv2) ($PROTOCOL_VERSION)
+  -m, --max-protocol-version="v4"                         Max protocol version supported by the backend cluster (default: v4, options: v3, v4, v5, DSEv1, DSEv2) ($MAX_PROTOCOL_VERSION)
+  -a, --bind=":9042"                                      Address to use to bind server ($BIND)
+  -f, --config=CONFIG                                     YAML configuration file ($CONFIG_FILE)
+      --debug                                             Show debug logging ($DEBUG)
+      --health-check                                      Enable liveness and readiness checks ($HEALTH_CHECK)
+      --http-bind=":8000"                                 Address to use to bind HTTP server used for health checks ($HTTP_BIND)
+      --heartbeat-interval=30s                            Interval between performing heartbeats to the cluster ($HEARTBEAT_INTERVAL)
+      --idle-timeout=60s                                  Duration between successful heartbeats before a connection to the cluster is considered unresponsive and closed ($IDLE_TIMEOUT)
+      --readiness-timeout=30s                             Duration the proxy is unable to connect to the backend cluster before it is considered not ready ($READINESS_TIMEOUT)
+      --num-conns=1                                       Number of connection to create to each node of the backend cluster ($NUM_CONNS)
+      --rpc-address=STRING                                Address to advertise in the 'system.local' table for 'rpc_address'. It must be set if configuring peer proxies ($RPC_ADDRESS)
+      --data-center=STRING                                Data center to use in system tables ($DATA_CENTER)
```

Comment:
We decided to use a different configuration approach for this but I think ours ("ZDM") is a bit more confusing when deploying "manually" by users instead of our automation. We might rethink this for v2.

We provide a single list of rpc_addresses to all proxies and a `PROXY_INDEX` that should be `0..n` where `n` is the total number of proxies. Then, each proxy grabs the rpc_address with its index from the list.